samples/petclinic-jpa/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.springframework.experimental</groupId>
-		<artifactId>spring-native-sample-parent</artifactId>
+		<artifactId>sprigit stng-native-sample-parent</artifactId>
```

Comment:
Pretty sure this is an accidental typing of a git command into your IDE window!

Reply:
Probably yes.